package.json

```diff
@@ -1,6 +1,6 @@
 {
   "scripts": {
-    "test": "eslint --ignore-pattern 'docs/**/*.js' --ignore-pattern 'staticfiles/**/*.js' --ignore-pattern 'env/**/*.js' . && stylint **/static/styles/"
+    "test": "eslint . && stylint **/static/styles/*.styl"
```

Comment:
Looks like the `*.styl` part got accidentally restored during the rebase. As per #165 we'll need to keep that out.